teradatalabs/iop-hive/Dockerfile

```diff
@@ -0,0 +1,67 @@
+# Copyright 2016 Teradata
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM teradatalabs/iop-base
+MAINTAINER Teradata Docker Team <docker@teradata.com>
+
+# INSTALL HADOOP AND HIVE
+RUN \
+  yum install -y \
+    zookeeper_4_2_0_0-server \
+  && ln -s /usr/iop/4.2.0.0/zookeeper/bin/zookeeper-server /usr/bin/zookeeper-server \
+  && yum -y clean all && rm -rf /tmp/* /var/tmp/*
+
+# INSTALL MYSQL (Hive Metastore)
+RUN yum install -y mysql-server mysql-connector-java \
```

Comment:
Mainly for consistency with other `*-hive` files and because sorting it out in all of them (and in all other *-master/slave images) is beyond the amount of work I want to spend on this... :)